From 1e51efbbcf09a1301873c8d8f896c3c5cd31a899 Mon Sep 17 00:00:00 2001
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Date: Mon, 15 Apr 2024 17:00:47 +0800
Subject: [PATCH 3/3] Update python versions in PYTHONPATH patch.

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 61e624d..f48113b 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -27,7 +27,7 @@ export DEB_HOST_MULTIARCH := $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 DEB_HOST_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 
 # Needed to bootstrap since the ros_workspace package does not yet exist.
-export PYTHONPATH=@(InstallationPrefix)/lib/python3.10/site-packages:@(InstallationPrefix)/lib/python3.9/site-packages
+export PYTHONPATH=@(InstallationPrefix)/lib/python3.12/site-packages:@(InstallationPrefix)/lib/python3.11/site-packages
 
 %:
 	dh $@@ -v --buildsystem=cmake --builddirectory=.obj-$(DEB_HOST_GNU_TYPE)
-- 
2.41.0

